lib/app/views/main_page.dart

```diff
@@ -113,7 +113,12 @@ class MainPage extends ConsumerWidget {
       return ref.watch(currentDeviceDataProvider).when(
             data: (data) {
               final app = ref.watch(currentAppProvider);
-              if (app.getAvailability(data) == Availability.unsupported) {
+              if (data.info.config.enabledCapabilities.isEmpty &&
```

Comment:
Why do you need the `data.info.config.enabledCapabilities.isEmpty` check? Is it not enough to simply check the name only?

Reply:
I decided to do it this way because I know what the `data` looks like for unrecognized devices, when that situation happens - it is constructed by [this code](https://github.com/Yubico/yubioath-flutter/pull/919/files#diff-1defc8f773208b04e321468e9a2c644111b109a46d1cbb344ccd314e020a3c78R304-R321)

Reply:
It might be cleaner to check `data.info.supportedCapabilities` since that data is static for a key, whereas `enabledCapabilities` can be modified by the user.

Reply:
I see - I will rather use supported capabilities, thanks